### PR TITLE
Update MIX2 roadmap

### PR DESCRIPTION
**Pull request by estark37**
* Discuss insecure downloads
* Add some notes about potential objections/concerns
* Rephrase the autoupgrading part slightly

### CODE REVIEW

**Comment by estark37** [comment]
@mikewest @jdeblasio wdyt?

File: proposed-level-2-roadmap.md [hunk 1]
```diff
@@ -52,6 +52,11 @@ remains a fairly blunt opt-in mechanism that developers have to know about and c
 Priming](https://wicg.github.io/hsts-priming/) aimed to bump the defaults up a bit, but hasn't shown
 enough value in based on the rumblings I've heard from Mozilla to be worth implementing.
 
+### Downloads
+
+Insecure downloads can be highly dangerous, but they are not treated as mixed content, and many
+browsers (including Chrome) do not even label them as insecure in the UI.
```
Comment (jdeblasio) [inline proposed-level-2-roadmap.md:58]: Is it worth additionally mentioning that Chrome doesn't (and _can't_) downgrade security indicators when insecure downloads may be present?

File: proposed-level-2-roadmap.md [hunk 2]
```diff
@@ -85,6 +90,10 @@ enough value in based on the rumblings I've heard from Mozilla to be worth imple
     removing it entirely, perhaps compromising based on use cases by moving it out of the address
     bar into developer tooling or extensions.
 
+5.  User agents should treat insecure downloads initiated from secure contexts as blockable mixed
+    content. To limit breakage, this measure may be restricted to high-risk downloads, e.g.,
+    executable file types.
```
Comment (jdeblasio) [inline proposed-level-2-roadmap.md:95]: If we're restricting when we block, is that really just saying we want it to be optionally blockable?

File: proposed-level-2-roadmap.md [hunk 3]
```diff
@@ -99,3 +108,16 @@ enough value in based on the rumblings I've heard from Mozilla to be worth imple
...
+
+1.  The proposals in this document could make it somewhat harder for developers to migrate to HTTPS,
+    because optionally-blockable mixed content and downloads would break if not upgradable. (In
+    contrast, those resources would continue to work during an HTTPS migration today, albeit with a
+    security indicator downgrade for optionally-blockable mixed content.) We feel that HTTPS
+    adoption has advanced sufficiently that it's now appropriate to start requiring stronger
+    configurations.
+
+2.  The downloads proposal above does not discourage insecure downloads initiated from insecure
+    contexts. Those should eventually be discouraged as well, but we suggest started with "mixed"
```
Comment (jdeblasio) [inline proposed-level-2-roadmap.md:122]: starting*

**Review by jdeblasio — COMMENTED** [review]
:tada:

**Comment by jdeblasio** [comment]
I'd love to get this old change merged.

**Comment by mikewest** [comment]
Merging, sorry I fell down on this. I'm adding you as a collaborator so you can feel empowered to land changes when I'm not paying attention for months. :(

**Comment by mikewest** [comment]
Actually, this is my repo. Can y'all move things to the w3c repo instead so we can eventually publish a draft?

**Comment by estark37** [comment]
Yeah, sorry for the delay here; now that we have a level 2 doc in the w3c repo, we should make this change there instead. I'll send a PR this morning.

**Pull request merged**